style: adjust add networks search results spacing

antonyayoub opened the pull request:
# Name of the feature/issue
Network search results spacing
## Description
Adjust networks search results spacing to be consistent with add token search spacing.
## Type of change

Click the correct/s option/s

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  This change requires a documentation update

julianariel reviewed (commented):
nice! looks much better now, but I think we can still improve some other things:

- There is a space between them that persists. It is caused by the **mt-1** class included in the `ChainDisplay` component. I think it would look better without it.

https://user-images.githubusercontent.com/11839151/194150438-7f03e4b3-6d6d-4da8-afae-0f59c2172cef.mp4

- Search bar and title have too much space between them, in this case, we have the add manually button but can still be reduced a bit. Also, the space between the title and the first result is a bit bigger than the one on AddTokens page.
- Would be great if we include the same scrollbar too.
![add tokens](https://user-images.githubusercontent.com/11839151/194150807-8b61c9f3-5df2-499f-88f2-123069ed13e1.png)

antonyayoub commented:
Great suggestions 🔥 
I adjusted them all. Have a look!

- Adjusted space between network results.
- Adjusted space between "Search Networks" and the actual results.
- Adjusted space between "Network not found" and "Search Networks".
- Added scroll bar to network results.

julianariel reviewed (approved):
LGTM, great work! 👏🏻
